minio.yaml

```diff
@@ -2,14 +2,19 @@ apiVersion: template.openshift.io/v1
 kind: Template
 labels:
   template: Minio
-message: Minio template
 metadata:
   annotations:
-    description: Deploys Minio using existing PVCs
+    description: This template deploys S3 compatible private Object Store Minio using existing PVCs.
+    tags: S3, Object Store
+    openshift.io/display-name: Minio Object Store
+    iconClass: icon-scala
```

Comment:
Maybe the icon could some generic storage icon?

Reply:
@juhanikataja  Do you have some other icon in mind? I couldn't find any other default icon

Reply:
Perhaps https://fontawesome.com/v4.7.0/icon/hdd-o:
```
iconClass: fa fa-hdd-o
```

Reply:
Or! `iconClass: fa fa-archive` (https://fontawesome.com/v4.7.0/icon/archive)

Reply:
fa fa-archive looks good, let's go with that